agent/game_service.py

```
@@ -0,0 +1,1131 @@
+        
+        return result
+    
+    finally:
+        session.close()
+
+
+@game_app.post("/marketplace/list")
+def list_toy(
+    request: ListToyRequest,
+    wallet_address: str,  # From auth
+    db: GameDatabase = Depends(get_db),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)